File: 0151-reverse-words-in-a-string/0151-reverse-words-in-a-string.java
class Solution {
    public String reverseWords(String s) {
        String[] arr = s.split(" ");
        String str = "";
        for(int i = arr.length - 1; i>=0; i--){
            if(arr[i] == "") continue;
            if(str.length() > 0) str += " ";
            str += arr[i];
        }
        return str;
    }
}

File: Leet Code/0151-reverse-words-in-a-string/0151-reverse-words-in-a-string.js
/**
 * @param {string} s
 * @return {string}
 */
var reverseWords = function(s) {
    s = s.split(' ')
    // let res = [];
    // for(let i=s.length-1; i>=0; i--){
    //     if(s[i] != ""){
    //         res.push(s[i])
    //     }
    // }
    // return res.join(' ')

    let str = ""

    for(let i=s.length-1; i>=0; i--){
        if(s[i] == "") continue;
        if(str.length>0) str += " "
        str += s[i]
    }
    return str;
};